hms-api/serverless.yml

```diff
@@ -141,7 +141,7 @@ functions:
     handler: src/handler/idea/list.list
     events:
       - http:
-          path: hackathon/{hackathonId}/ideas
+          path: hackathon/{id}/ideas
```

Comment:
this will certainly break tests, and I would actually say to go in the opposite direction and use the longer names for everything if possible.  It's fine like this though, but fix the tests :)

Reply:
Cloudformation would not accept the longer name though, if there is another path with `hackathon/{id}*`. That was the issue with deployment.

Reply:
weird